File: ASP.NET-Backend/ASP.NET-Backend/Models/Salon.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ASP.NET_Backend.Models
{
    public class Salon
    {
        public int salonId { get; set; }
        public string name { get; set; }
        public string address { get; set; }
        public string zipCode { get; set; }
        public string city { get; set; }
        public string phone { get; set; }
        public string email { get; set; }
        public int salonBookingId { get; set; }
    }
}

File: NodeBackend/Models/Salon.js
module.exports = class Salon {

    constructor(salonId, name, address, zipCode, city, phone, email, salonBookingId) {
        this.salonId = salonId;
        this.name = name;
        this.address = address;
        this.zipCode = zipCode;
        this.city = city;
        this.phone = phone;
        this.email = email;
        this.salonBookingId = salonBookingId;

    }
}